commands/create.py

```diff
@@ -18,28 +19,31 @@ class FmCreaterCommand(FmWindowCommand):
     final ones if it doesn't exists. Finaly, opens the file"""
 
     def run(self, abspath, input_path):
-        input_path = user_friendly(input_path)
```

Comment:
> We shouldn't copy paste the code from bracex into this repo, we should just add it as a dependency of this package (https://github.com/wbond/package_control_channel/blob/master/repository/dependencies.json#L95-L109)

I did so mainly because I forgot we added bracex to the dependencies :)

I’ll update

Reply:
@math2001 What do you mean by  `this user_friendly call has been thrown away.`